opm/autodiff/BlackoilSolventModel_impl.hpp

```diff
@@ -224,9 +227,18 @@ namespace Opm {
 
     }
 
-
-
-
+    template <class Grid>
+    void
+    BlackoilSolventModel<Grid>::updateEquationsScaling()
+    {
+        Base::updateEquationsScaling();
+        assert(MaxNumPhases + 1 == resdual.matbalscale.size());
```

Comment:
> This cannot compile (typo in variable name).

Technically, it **will** compile if `NDEBUG` is defined as the misprint only occurs within an `assert`.  Of course, that does not mean that it should not be fixed.